stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -155,18 +155,15 @@ public interface ChatDomain {
             return this
         }
 
-        // TODO: do we even need this, or is it better to have it at the low level client?
+        @Deprecated("This method is deprecated, you should configure it into the ChatClient")
         public fun notificationConfig(notificationConfig: NotificationConfig): Builder {
-            this.notificationConfig = notificationConfig
+            ChatLogger.get("ChatDomain").logE("This method is deprecated, you should configure it into the ChatClient")
```

Comment:
This method can still be called, but it's now a silent no-op, which will be unexpected for clients. Perhaps we want to mark this as `level = ERROR` so that it breaks the build? And even then, I'd keep it functional if possible, until we remove it.

Reply:
Yep, `level = ERROR` is the best way for now. Keep old behavior could give us some problem with a bad configuration

Reply:
We might still have people who already have compiled code that calls into this method, but I guess it's gonna be rare. Maybe throw an exception for that case, so that this never gets silently executed?

Reply:
:+1: